src/lib/structures/game/Game.ts

```diff
@@ -103,11 +98,15 @@ export class Game {
 	}
 
 	public start(interaction: CommandInteraction) {
-		const answerType = [AcceptedAnswer.Song, AcceptedAnswer.Artist].includes(this.acceptedAnswer)
-			? this.acceptedAnswer.toLowerCase()
-			: `song ${italic(this.acceptedAnswer === AcceptedAnswer.Both ? 'and' : 'or')} artist`;
+		let answerTypeString = '';
+		if ([AcceptedAnswer.Song, AcceptedAnswer.Artist].includes(this.acceptedAnswer)) {
+			answerTypeString = this.acceptedAnswer.toLowerCase();
```

Comment:
It will always be in lowercase
```suggestion
			answerTypeString = this.acceptedAnswer;
```